recipes-bsp/barebox/barebox.inc

```diff
@@ -94,11 +94,9 @@ do_deploy () {
 		install -m 644 -t ${DEPLOYDIR}/ ${B}/barebox.efi
 	fi
 	for IMAGE in ${B}/images/*.img; do
-		if [ -e ${IMAGE} ]; then
```

Comment:
Why can't it be empty? If you e.g. compile barebox for efi, images is empty AFAIR.

Reply:
Ah, I see now, the glob wouldn't expand in that case. Removing the check is correct. That way we get error messages later on if `BAREBOX_IMAGES` contains a non-existent file.

Reply:
Arg.. I see what you mean. Didn't checked the efi-only case. IMHO we should fail if the someone specifies an invalid image name instead of ignoring/skipping that image but you're right with efi case.

Reply:
Ah, right. `shopt -s nullglob` is not the default. What kind of shell is used with bitbake?

Reply:
Unfortunately.. Bitbake uses `bin/sh` so the answer is it depends